snapcraft/extensions/gnome.py

```diff
@@ -147,6 +147,7 @@ def get_root_snippet(self) -> Dict[str, Any]:
             "environment": {
                 "SNAP_DESKTOP_RUNTIME": "$SNAP/gnome-platform",
                 "GTK_USE_PORTAL": "1",
+                "GDK_DEBUG": "portals",
```

Comment:
I'd separate this as it's unrelated to the other changes in the PR.  Also, I'm not confident of the implications we'll have across the wide range of snaps we have using the extension, so I'd say it's a pretty risky change.